### PR TITLE
OSDOCS-12239: 4.18 Release notes file template

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.18

Issue:
[OSDOCS-12239](https://issues.redhat.com/browse/OSDOCS-12239)

Link to docs preview:
* [RN 4.18 doc](https://83073--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-18-release-notes.html)
* [Differences between OpenShift Container Platform 3 and 4](https://83073--ocpdocs-pr.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/planning-migration-3-4.html)

### CODE REVIEW

File: release_notes/ocp-4-18-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,1677 @@
...
+[id="ocp-release-notes-networking_{context}"]
+=== Networking
+
+[id="ocp-release-notes-nodes_{context}"]
+=== Nodes
+
+[id="ocp-release-notes-postinstallation-configuration_{context}"]
+=== Postinstallation configuration
+
+[id="ocp-release-notes-registry"]
```
Comment (skopacz1) [inline release_notes/ocp-4-18-release-notes.adoc:445]: This doesn't have to be done in this PR if it'll get in the way too much, but please comb through this assembly at some point and add contexts to any IDs that are missing them:
```suggestion
[id="ocp-release-notes-registry_{context}"]
```

**Review by skopacz1 — APPROVED** [review]
One consideration for this PR or a later one, otherwise LGTM!

File: release_notes/ocp-4-18-release-notes.adoc (hunk 1 above)
Comment (dfitzmau) [reply release_notes/ocp-4-18-release-notes.adoc]: Thanks. I updated three instances!

**Pull request merged**